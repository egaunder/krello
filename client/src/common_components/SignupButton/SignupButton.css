.signup__button {
  outline: none;
  border: none;
  background-color: #f1c40f;
  border-radius: 3px;
  line-height: 1.5em;
  font-size: 1em;
  color: white; }
  .signup__button:hover {
    transition-duration: 200ms;
    background-color: #f39c12; }
