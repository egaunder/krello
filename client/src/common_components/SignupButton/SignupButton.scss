@import '../../utils/styles/common_colors.scss';

.signup__button {
  outline: none;
  border: none;
  background-color: $sunflower;
  border-radius: 3px;
  line-height: 1.5em;
  font-size: 1em;
  color: white;
  &:hover {
    transition-duration: 200ms;
    background-color: $orange;
  }
}