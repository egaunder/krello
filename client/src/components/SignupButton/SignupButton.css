#signup-button {
  background-color: #61bd4f;
  color: white; }
  #signup-button:hover {
    background-color: #4F992D; }
