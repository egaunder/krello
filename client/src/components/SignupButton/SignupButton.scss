#signup-button {
  background-color: #61bd4f;
  color: white;
  
  &:hover {
    background-color: #4F992D;
  }
}