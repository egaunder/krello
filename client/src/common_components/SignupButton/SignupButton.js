import React from 'react'
import PropTypes from 'prop-types'
import './SignupButton.css'

const SignupButton = props => (
  <button className="signup__button" onClick={() => props.onClick()}>Signup</button>
)

SignupButton.propTypes = { onClick: PropTypes.func.isRequired }

export default SignupButton
