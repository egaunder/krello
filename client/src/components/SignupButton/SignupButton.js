import React from 'react'
import './SignupButton.css'

const SignupButton = () => (
  <button id="signup-button">
    Signup
  </button>
)

export default SignupButton
