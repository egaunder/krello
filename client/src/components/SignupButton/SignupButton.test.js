import React from 'react'
import { shallow } from 'enzyme'
import { shallowToJson } from 'enzyme-to-json'
import SignupButton from './SignupButton'

describe('[SignupButton]', () => {
  test('should render correctly', () => {
    const output = shallow(<SignupButton />)
    expect(shallowToJson(output)).toMatchSnapshot()
  })
})
