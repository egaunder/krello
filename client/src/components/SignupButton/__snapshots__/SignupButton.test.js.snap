// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[SignupButton] should render correctly 1`] = `
<button
  id="signup-button"
>
  Signup
</button>
`;
